src/Simplex_tree/include/gudhi/Simplex_tree.h

```diff
@@ -2595,17 +2676,15 @@ class Simplex_tree {
     if (members_size > 0) {
       if constexpr (!Options::stable_simplex_handles) sib->members_.reserve(members_size);
       Vertex_handle vertex;
-      Filtration_value filtration;
+      Filtration_value filtration(0);
```

Comment:
Why?

Reply:
I see. Yes, an empty vector is good, since the only thing we will do with it is overwrite it. That constructor is confusing though. When I see `Filtration_value(n)`, I don't really expect {-inf,-inf,...,-inf} n times. For plain `double`, `Filtration_value filtration;` (without `(0)`) is nicer because it gives sanitizers a chance to warn if there is a case where we forget to assign a value.
But ok for now, we can rediscuss that in the multiparameter branch.
Actually, this seems to be in the deserialization function, does that work in the multiparameter branch?

Reply:
I remember now another reason why I did this. Sometimes `filtration` does not get initialized by default with 0 exactly, but with just a very small value close to 0. And then, the constructor of `Filtration_simplex_base_dummy` complains and throws in Debug mode.

> Actually, this seems to be in the deserialization function, does that work in the multiparameter branch?

I naively though that it does as the methods looked quite general, but it does not... So I added some methods to the FiltrationValue concept (saying that here are only necessary in the case of de/serialization) and also some tests in `simplex_tree_serialization_unit_test.cpp` to make it work.

Reply:
I added a comment and also added a constructor to the concept. Does it look fine?

I also added `Gudhi::simplex_tree::serialize_trivial` etc. in the simplex_tree group, because otherwise, they didn't appear in the documentation for some reason.